src/Controls/samples/Controls.Sample.SingleProject/Startup.cs

```diff
@@ -1,18 +1,43 @@
-﻿using Microsoft.Maui;
+﻿using System;
+using Microsoft.Maui;
 using Microsoft.Maui.Controls.Compatibility;
 using Microsoft.Maui.Hosting;
 using Microsoft.AspNetCore.Components.WebView.Maui;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
 
 namespace Maui.Controls.Sample.SingleProject
 {
 	public class Startup : IStartup
 	{
+		internal static bool UseBlazor = false;
+
 		public void Configure(IAppHostBuilder appBuilder)
 		{
 			appBuilder
 				.UseFormsCompatibility()
 				.RegisterBlazorMauiWebView()
 				.UseMauiApp<MyApp>();
+
+			if (UseBlazor)
+            {
```

Comment:
There are some pretty eclectic variations in indentation across different files in this PR. Is it because the MAUI repo as a whole has differing conventions in different places?

But this `{` in particular is a true maverick.

Reply:
The MAUI bot will fix my mess later 😄